interfaces/builtin/account_control.go

```diff
@@ -60,8 +60,8 @@ capability fsetid,
 const accountControlConnectedPlugSecComp = `
 # useradd requires chowning to 'shadow'
 # TODO: dynamically determine the shadow gid to support alternate cores
-fchown - 0 42
-fchown32 - 0 42
+fchown - root shadow
```

Comment:
Interestingly this will start failing unless user lookup is done inside the mount namespace with /etc/group from the core snap. On my suse box I see this:

```
cat /etc/group | grep shadow
shadow:x:15:
```
But compare with:
```
cat /snap/core/current/etc/group | grep shadow
shadow:x:42:
```

So -1 unless we ensure this is done correctly.

Reply:
But on classic /etc is the /etc from the host, not the core snap, so we should always have the same permissions (in fact, this change is precisely to address the issue with suse and us hardcoding 42).

That said, I'll need to look at the testsuite to fix TestCompile on suse.

Reply:
At runtime, a strictly confined snap will fail to `chown shdow ...` because shadow will resolve to 42 but the generated seccomp profile, compiled from the "outside" will allow 15. What am I missing in my reasoning?

Reply:
We discussed this on  IRC (my previous comment stands).